packages/@aws-cdk/aws-apigatewayv2-authorizers-alpha/test/http/integ.lambda.ts

```diff
@@ -29,6 +27,17 @@ const authorizer = new HttpLambdaAuthorizer('LambdaAuthorizer', authHandler, {
   responseTypes: [HttpLambdaResponseType.SIMPLE],
 });
 
+const defaultAuthorizer = new HttpLambdaAuthorizer('LambdaDefaultAuthorizer', authHandler, {
+  authorizerName: 'my-simple-authorizer',
+  identitySource: ['$request.header.X-API-Key'],
+  responseTypes: [HttpLambdaResponseType.SIMPLE],
+});
+
+const httpApi = new HttpApi(stack, 'MyHttpApi');
+const httpApiWithDefaultAuthorizer = new HttpApi(stack, 'MyHttpApiWithDefaultAuthorizer', {
+  defaultAuthorizer,
+});
```

Comment:
Can you also add `defaultAuthorizationScopes` to the `httpApiWithDefaultAuthorizer` API?

Reply:
https://github.com/aws/aws-cdk/pull/27576/commits/41ca1bcdf3c0965351aa3cc8742ee1cab7876011

Reply:
https://github.com/aws/aws-cdk/pull/27576/commits/2a76067a772efa617da5b212c557ca513247b742#diff-a085a7249a4f93dd2c9dd99905d4c1051f3201b48a2afd5a9a211d984e79c45aR37

Reply:
I changed integ.user-pool instead of integ.lambda because defaultAuthorizationScopes (Authorization Scopes) are only valid for COGNITO_USER_POOLS and JWT authorization types

https://github.com/aws/aws-cdk/pull/27576/commits/e3e3f9ec93532cae32ee408c4f8973cb78632d36

Reply:
👍 Thanks for the clarification.